app/models/index.js

```diff
@@ -521,6 +521,12 @@ templateAppendix.belongsTo(project, {
   as: 'project', foreignKey: 'projectId', targetKey: 'id', onDelete: 'CASCADE', constraints: true,
 });
 
+const templateExtendedScope = {
+  attributes: {exclude: ['id', 'deletedAt', 'updatedBy']},
+  include: [],
+};
+templateAppendix.addScope('extended', templateExtendedScope);
```

Comment:
we usually define scope in the model file, see line 35 of app/models/template/templateAppendix.js. Move scope definition to there